keyserver/README.md

```diff
@@ -0,0 +1,35 @@
+# CONIKS Server implementation in Golang
+
+## Usage
+```
+⇒  go install github.com/coniks-sys/coniks-go/keyserver/coniksserver
+⇒  coniksserver -h
+Usage of coniksserver:
+  -config string
+    	Path to the configuration file (default "config.toml")
+  -dir string
+    	Directory location for storing generated files (default ".")
+  -genconfig
+    	Generate server configurations file
+  -genkey
+    	Generate signing/VRF key, e.g. -genkey=sign,vrf
+```
+Generate a private key for secure connection (TLS)
+```
+⇒  openssl ecparam -genkey -name prime256v1 -out server.key
+```
+
+Generation of self-signed(x509) public key (PEM-encodings `.pem`) based on the private (`.key`)
+```
+⇒  openssl req -new -x509 -sha256 -key server.key -out server.pem -days 3650
+```
+
+Run the server
+```
+⇒  mkdir coniks/
+⇒  coniksserver -dir coniks -genkey -genconfig
+⇒  coniksserver -dir coniks  // run
```

Comment:
Here is yet another (late) proposal: Wouldn't it be nice if for instance `coniksserver init` created all necessary files/generates all keys in the current working directory (or in a default directory). Then above commands could boil down to:
```
 ⇒  mkdir coniks; cd coniks
 ⇒  coniksserver init // creates all files, maybe even the self-signed ssl keys/cert with sane defaults
 ⇒  coniksserver run // looks for config files wit default names in the current directory if not specified differently
``` 
Just another idea. I like how for instance `git` or `go` use "subcommands" to make the commandline tools better structured and hence easier to use. But I also like what we have now.